### ISSUE TITLE
`plotSimulatedVsObserved` y and x axis limits should be identical

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite-R#1051]
![](https://i.imgur.com/J6iz18f.png)

**Comment by PavelBal** [comment]
Such plots look odd if the axes have different limits. The user expects the identity line to go through bottom left corner to top right.

**Comment by IndrajeetPatil** [comment]
After fixing #1052, this looks much better, although they don't have the exact same axes labels. One terminates at 10, while the other at 100.

``` r
library(ospsuite)
#> Loading required package: rClr
#> Loading the dynamic library for Microsoft .NET runtime...
#> Loaded Common Language Runtime version 4.0.30319.42000

simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
sim <- loadSimulation(simFilePath)

simResults <- runSimulation(sim)

obsData <- lapply(c("ObsDataAciclovir_1.pkml", "ObsDataAciclovir_2.pkml", "ObsDataAciclovir_3.pkml"),
                  \(x){loadDataSetFromPKML(system.file("extdata", x, package = "ospsuite"))})

names(obsData) <- lapply(obsData, \(x){x$name})

outputPath <- "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)"
myDataCombined <- DataCombined$new()
# Add simulated results
myDataCombined$addSimulationResults(simulationResults = simResults,
                                    quantitiesOrPaths = outputPath,
                                    groups = "Aciclovir PVB")

# Add observed data set
myDataCombined$addDataSets(obsData$`Vergin 1995.Iv`, groups = "Aciclovir PVB")

plotObservedVsSimulated(myDataCombined)
```

![](https://i.imgur.com/cDYAwuv.png)

<sup>Created on 2022-08-04 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

**Comment by PavelBal** [comment]
By default, it should look like this:

![image](https://user-images.githubusercontent.com/25903040/183440919-080aee9f-eb9f-46e2-bd3f-9117b45f35cc.png)

**Comment by IndrajeetPatil** [comment]
> By default, it should look like this:

@pchelle In https://github.com/Open-Systems-Pharmacology/TLF-Library/issues/350, you had mentioned that this is better handled in the `{ospsuite}` plotting function itself. But I see no context in which `{tlf}` shouldn't be producing similar axes for this particular plot, since the values on both axes are paired. WDYT?

**Comment by IndrajeetPatil** [comment]
This should be fixed in PR #1054. Thanks, @pchelle!

  ``` r
library(ospsuite)

simFilePath <- system.file("extdata", "Aciclovir.pkml", package = "ospsuite")
sim <- loadSimulation(simFilePath)

simResults <- runSimulation(sim)

obsData <- lapply(c("ObsDataAciclovir_1.pkml", "ObsDataAciclovir_2.pkml", "ObsDataAciclovir_3.pkml"),
                  \(x){loadDataSetFromPKML(system.file("extdata", x, package = "ospsuite"))})

names(obsData) <- lapply(obsData, \(x){x$name})

outputPath <- "Organism|PeripheralVenousBlood|Aciclovir|Plasma (Peripheral Venous Blood)"
myDataCombined <- DataCombined$new()
# Add simulated results
myDataCombined$addSimulationResults(simulationResults = simResults,
                                    quantitiesOrPaths = outputPath,
                                    groups = "Aciclovir PVB")


# Add observed data set
myDataCombined$addDataSets(obsData$`Vergin 1995.Iv`, groups = "Aciclovir PVB")

plotObservedVsSimulated(myDataCombined)
```

![](https://i.imgur.com/7BfeN97.png)

<sup>Created on 2022-08-11 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>
